lb_content_resolver/model/recording.py

```diff
@@ -12,7 +34,8 @@ class Meta:
         database = db
 
     id = AutoField()
-    file_path = TextField(null=False, unique=True)
+    file_id = TextField(null=False, unique=True)
+    file_id_type = FileIdTypeField(null=False)
```

Comment:
I think we have to create few new indexes, one for uniqueness on both fields, and one for each field:

```diff
diff --git a/lb_content_resolver/model/recording.py b/lb_content_resolver/model/recording.py
index 2512aa7..713c570 100644
--- a/lb_content_resolver/model/recording.py
+++ b/lb_content_resolver/model/recording.py
@@ -32,10 +32,14 @@ class Recording(Model):
 
     class Meta:
         database = db
+        indexes = (
+            # create a unique on (file_id, file_id_type)
+            (('file_id', 'file_id_type'), True),
+        )
 
     id = AutoField()
-    file_id = TextField(null=False, unique=True)
-    file_id_type = FileIdTypeField(null=False)
+    file_id = TextField(null=False, index=True)
+    file_id_type = FileIdTypeField(null=False, index=True)
     mtime = TimestampField(null=False)
 
     artist_name = TextField(null=True)
```

Reply:
Good call!

Reply:
Applied.